types/wordpress__blocks/api/registration.d.ts

```diff
@@ -191,14 +191,14 @@ export function getBlockVariations(blockName: string, scope?: BlockVariationScop
  * Registers one or more new block variations for the given block type.
  *
  * @param blockName - Name of the block (example: “core/columns”).
- * @param variation - Object or array of objects describing block variations.
+ * @param variation - Variation configuration object (or array of if more than one).
  */
 export function registerBlockVariation(blockName: string, variation: BlockVariation | BlockVariation[]): void;
 
 /**
  * Unregisters one or more variations defined for the given block type.
  *
  * @param blockName - Name of the block (example: “core/columns”).
- * @param variationName - Name or array of variation names defined for the block.
+ * @param variationName - Variation name defined for the block (or array of if more than one).
```

Comment:
it's the "defined for the block" that's really throwing me off. We're wanting to unregister a variation, but not a block.

"Name of registered variation"

"Variation name"

Maybe it would help to discuss why "defined for the block" is relevant here as I might be missing something obvious to others.

Reply:
haha felt the same but just carried it over from Gutenberg repo - if we share the sentiment here there must be something to it right? I'll update it quickly

Reply:
but then.. `variationName - Variation name (or array...` the param name is self explanatory

Reply:
> the param name is self explanatory

This is one of the reasons I like describing purpose of parameters rather than essence.

"Unregister this variation (or array if unregistering multiple variations)"

Reply:
What do you think "Name of variation to be unregistered (or array if unregistering multiple variations)"?

We'll have round two when suggesting these JSDoc changes on Gutenberg side haha

Reply:
that sounds good!